common/web/keyboard-processor/tests/dom/web-test-runner.config.mjs

```diff
@@ -21,7 +21,7 @@ export default {
   concurrency: 10,
   nodeResolve: true,
   files: [
-    '**/*.spec.ts'
+    'build/lib/**/*.spec.mjs'
```

Comment:
I would strongly prefer a specific build folder for tests, then a specific folder for each test set.

Something like `build/test/dom/**/*.spec.mjs` - this pattern is being maintained for `/web/src/test/auto/dom/` tests, so why not be consistent with the pattern and reuse it here?

(While this module's headless tests aren't yet in TS, they could then go under `build/test/headless/**`.)

A general pattern is for `lib` folders to hold bundled / fully-compiled outputs, not testing files... right?  Or am I just mistaken about that?  Either way, that's a pattern I've held to strongly with the lower-level packages; test file outputs do not belong in that folder, in my opinion.  Too messy.

Reply:
Done.